backend/protos/mgmt/v1alpha1/transformer.proto

```diff
@@ -301,7 +301,9 @@ message GenerateSha256Hash {}
 
 message GenerateSSN {}
 
-message GenerateState {}
+message GenerateState {
+  bool state_code = 1;
```

Comment:
This could use a better name besides just `state_code` as that isn't super clear what it means.

What does it mean if it's true, what does it mean if it's false? 
Please also add a comment here describing what this property does.

I'm assuming that the `false` value also retains the previous empty value?

Reply:
im going to swap it actually for `generate_full_name` that way it maintains the current logic where if it's false (which is the default) it'll return the state code, otherwise users can opt into returning the full state name